utils/frame/benchmarking-cli/src/lib.rs

```diff
@@ -68,9 +68,9 @@ pub struct BenchmarkCmd {
 	#[structopt(long)]
 	pub header: Option<std::path::PathBuf>,
 
-	/// Output the trait definition to a Rust file.
+	/// Path to Handlebars template file used for outputting benchmark results. (Optional)
 	#[structopt(long)]
-	pub trait_def: bool,
+	pub template: Option<std::path::PathBuf>,
```

Comment:
Shouldn't we the default value feature here to make the default template discoverable on the CLI?